hangman.c

```diff
@@ -0,0 +1,312 @@
+  scanf("%c",&guess6);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1;
+    if(guess6 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly!\n");
+      errors--;
+      break;
+    }
+  }
+
+  if(errors < 1){
+    printf("You lose!");
+    exit();
+  }
+
+  printIncorrectLetters();
+  printRemainingLetters();
+  printf("Please input your seventh guess.\n")
+
+  char guess7;
+  scanf("%c",&guess7);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1;
+    if(guess7 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly!\n");
+      errors--;
+      break;
+    }
+  }
+
+
+  if(errors < 1){
+    printf("You lose!");
+    exit();
+  }
+
+  printIncorrectLetters();
+  printRemainingLetters();
+  printf("Please input your eigth guess.\n");
+
+  char guess8;
+  scanf("%c",&guess8);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1;
+    if(guess8 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly!\n");
+      errors--;
+      break;
+    }
+  }
+
+
+  if(errors < 1){
+    printf("You lose!");
+    exit();
+  }
+
+  printIncorrectLetters();
+  printRemainingLetters();
+  printf("Please input your ninth guess.\n");
+
+  char guess6;
+  scanf("%c",&guess9);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1;
+    if(guess9 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly!\n");
+      errors--;
+      break;
+    }
+  }
+
+
+  if(errors < 1){
+    printf("You lose!");
+    exit();
+  }
+
+  printIncorrectLetters();
+  printRemainingLetters();
+  printf("Please input your tenth guess.\n");
+
+  char guess10;
+  scanf("%c",&guess10);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1;
+    if(guess10 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly!\n");
+      errors--;
+      break;
+    }
+  }
+
+
+  if(errors < 1){
+    printf("You lose!");
+    exit();
+  }
+
+  printIncorrectLetters();
+  printRemainingLetters();
+  printf("Please input your eleventh guess.\n");
+
+  char guess11;
+  scanf("%c",&guess11);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1;
+    if(guess11 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly!\n");
+      errors--;
+      break;
+    }
+  }
+
+
+  if(error < 1){
+    printf("You lose!");
+    exit();
+  }
+
+  printIncorrectLetters();
+  printRemainingLetters();
+  printf("Please input your twelfth guess.\n");
+
+  char guess12;
+  scanf("%c",&guess12);
+
+  for(int i = 0; i < 9; i++){
+    int x = 1;
+    if(guess12 == answer[i]){
+      guess[i] = answer[i];
+      x = 0;
+    }
+    if(x == 0){
+      printf("You guessed incorrectly!\n");
+      errors--;
+      break;
+    }
+  }
+
+
+  void printRemainingLetters{
```

Comment:
methods shouldn't be inside the main - they should be completely outside so that you can call them. This is a huge part of the problem with this code.